.circleci/config.yml

```diff
@@ -410,10 +410,14 @@ workflows:
     jobs:
       - build:
           filters:
+            branches:
+              ignore: /tests.*/
```

Comment:
```suggestion
              ignore: /^tests\//
```
might be safer? right now it'll match any branch with `tests` in the name

Reply:
Actually, the `^` is not needed here. The current regex matches branches that start with `tests` (I've tested the behaviour on CircleCI).

I agree that adding `\/` makes sense. I'll do that now.

Reply:
Updated now.

I have verified that branches containing `tests/` not at the start are not ignored by this.

Reply:
interesting! Thanks for testing.